README.md

```diff
@@ -29,9 +29,9 @@
 - Generates a comprehensive data dictionary of your source and modeled Ad Reporting data via the [dbt docs site](https://fivetran.github.io/dbt_ad_reporting/)
 
 <!--section="ad_reporting_transformation_model"-->
-Refer to the table below for a detailed view of final models materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models. 
+Refer to the table below for a detailed view of final tables materialized by default within this package. Additionally, check out our [Docs site](https://fivetran.github.io/dbt_ad_reporting/#!/overview) for more details about these models.
```

Comment:
I see that model is renamed to table in this section but in no other sections, is this the only section in which these updates are needed?

Reply:
Yes this is the only section since this is what is displayed in the UI